### PR TITLE
PR: Tidy `README.md` issue #69

### PR DESCRIPTION
**Pull request by nelsonic**
This maintenance PR tidies the `README.md` to focus on the content. 🧹 
Also releases a new version to `Hex.pm` and `NPM` with the latest quotes. 📦 
@iteles thanks for reviewing/merging. 🙏

### CODE REVIEW

File: quotes.json [hunk 1]
```diff
@@ -209,9 +209,9 @@
   },
   {
     "author": "Albert Gray",
-    "text": "Winners have simply formed the habit of doing things losers don't like to do.",
-    "source" : "https://www.goodreads.com/quotes/524224",
-    "tags" : "winners, habit, losers"
+    "source": "https://www.goodreads.com/quotes/524224",
```
Comment (nelsonic) [inline quotes.json:212]: these updates were made by `JSONLint` ... I think they are good so keeping.

**Comment by nelsonic** [comment]
@iteles this mini PR does some tidy-up I spotted. Hopefully it's a quick read. Thanks.

File: README.md [hunk 2]
```diff
@@ -71,7 +51,9 @@ The
 file
 can be used in _any_ project or programming language.
 
-The module can be used from `Elixir` and `Javascript`/`Node.js`.
+The module can be used from `Elixir` and `JavaScript` (`Node.js`, `TypeScript`, etc.) <br />
+(_if you want to use it in a different language
+[**please tell us**](https://github.com/dwyl/quotes/issues/6)_)
```
Comment (iteles) [inline README.md:56]: 👍

File: README.md [hunk 3]
```diff
@@ -299,14 +282,12 @@ there simply isn't any _empirical **evidence**_.
...
 
-
-
 ### _Improve_ Existing Quotes with Sources & Tags
 
 Open the `quotes.json` file and scroll through the quotes. <br />
 If you find a quote that you like
 (_that doesn't already have "source" or "tags")
-you can _enhance_ it with a source and tags.
+you can \_enhance_ it with a source and tags.
```
Comment (iteles) [inline README.md:290]: Not sure why `\` is useful here but no harm no foul 👍

File: README.md [hunk 4]
```diff
@@ -526,3 +501,45 @@ see: `Utils.clean` in
...
+please make sure you
+[read the `Contributing` section](https://github.com/dwyl/quotes#contributing)
+of this repo.
+
+There are many websites that already contain quotes
+and this repository is
+intended as a _subset_ of uniquely _inspiring_ quotes.
+See [`Avoid Quotes That Don't Inspire Action`](https://github.com/dwyl/quotes#avoid-quotes-that-dont-inspire-action).
+
+As such, **_we remind you that we will only be able to accept PRs that conform
```
Comment (iteles) [inline README.md:517]: YES. This section is SO necessary ❤️  People have just been using this repo as a green squares machine during hacktoberfest 🙄 
Thank you for adding it.

File: README.md [hunk 5]
```diff
@@ -526,3 +501,45 @@ see: `Utils.clean` in
...
+
+As noted above,
+this repo focusses on quotes
+that **_unequivocally_ inspire action**.
+
+If a quote just makes people think,
+but doesn't inspire action we may not merge your Pull Request ... 😞
+
+Please don't be personally offended.
+You're very welcome to **_fork_** the project
```
Comment (iteles) [inline README.md:541]: 👍 🍴

**Review by iteles — APPROVED** [review]
Very necessary addition of the disclaimer 🎉 
Thank you!

**Pull request merged**